app/helpers/application_helper/button/generic_object_definition_delete_button.rb

```diff
@@ -0,0 +1,17 @@
+class ApplicationHelper::Button::GenericObjectDefinitionDeleteButton < ApplicationHelper::Button::Basic
+  def visible?
+    if @display == 'generic_objects'
+      false
+    else
+      true
+    end
+  end
+
+  def disabled?
+    if @record.generic_objects.count > 0
```

Comment:
def disabled?
  @record.generic_objects.count.zero?
end

Reply:
Done in 446f7c4